packages/cli/src/zosfiles/copy/ds/Ds.handler.ts

```diff
@@ -21,7 +21,7 @@ export default class DsHandler extends ZosFilesBaseHandler {
     public async processWithSession(commandParameters: IHandlerParameters, session: AbstractSession): Promise<IZosFilesResponse> {
         const fromDataSet: IDataSet = getDataSet(commandParameters.arguments.fromDataSetName);
         const toDataSet: IDataSet = getDataSet(commandParameters.arguments.toDataSetName);
-        const options: ICopyDatasetOptions = { fromDataSet, responseTimeout: commandParameters.arguments.responseTimeout };
+        const options: ICopyDatasetOptions = { "from-dataset": fromDataSet, responseTimeout: commandParameters.arguments.responseTimeout };
```

Comment:
This change seems.... weird.

Reply:
the git history for this indicated that a person made this change in `next` a long time ago to get our response objects to  match with the z/osmf payloads.  it's a breaking change so it was in `next` and not in `master`.  

i kept it bc it was previously approved / merged in there.  do you think it's ok to keep?

Reply:
I think it should be okay to keep. I was just a little confused, because I don't think we have any other object with a property that needs to be encapsulated in quotes and, in some cases, brackets. Just seemed a little out of place.

Reply:
alright - we'll keep this as is and fixed later if needed